packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -181,6 +199,10 @@ export class HbarLimitService implements IHbarLimitService {
     requestDetails: RequestDetails,
     estimatedTxFee: number = 0,
   ): Promise<boolean> {
+    if (!this.isEnabled()) {
+      return false;
+    }
```

Comment:
Please, add this check to `addExpense` as well:
```typescript
async addExpense(cost: number, ethAddress: string, requestDetails: RequestDetails): Promise<void> {
  if (!this.isEnabled()) {
    return;
  }
  ...
}
```

Reply:
Done.